htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -111,7 +111,7 @@ class CommandeFournisseur extends CommonOrder
 
 	public $socid;
 	public $fourn_id;
-	public $date;
+	public $date_creation;
```

Comment:
We must keep the field $date (date of order) that may be different than date_creation.
I suggest to add public $date_creation instead of just replacing date.